src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -375,24 +375,31 @@ namespace SurfaceGeometry {
         state.dataSurface->ShadingSurfaceFirst = -1;
         state.dataSurface->ShadingSurfaceLast = -1;
 
+        // Reserve space to avoid excess allocations
+        state.dataSurface->AllExtSolAndShadingSurfaceList.reserve(state.dataSurface->TotSurfaces);
```

Comment:
Add `.reserve` to avoid reduce the hit count on `malloc`

Reply:
:+1: :+1: